larpandoracontent/LArHelpers/LArPfoHelper.cc

```diff
@@ -173,6 +173,38 @@ void LArPfoHelper::GetAllDownstreamPfos(const ParticleFlowObject *const pPfo, Pf
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void LArPfoHelper::GetAllDownstreamPfos(
+    const pandora::ParticleFlowObject *const pPfo, pandora::PfoList &outputTrackPfoList, pandora::PfoList &outputLeadingShowerPfoList)
+{
+    if (LArPfoHelper::IsTrack(pPfo))
+    {
+        outputTrackPfoList.emplace_back(pPfo);
+        const PfoList &childPfoList(pPfo->GetDaughterPfoList());
+        for (const ParticleFlowObject *pChild : childPfoList)
+        {
+            if (std::find(outputTrackPfoList.begin(), outputTrackPfoList.end(), pChild) == outputTrackPfoList.end())
+            {
+                const int pdg{std::abs(pChild->GetParticleId())};
+                if (pdg == E_MINUS)
```

Comment:
Probably missing something, but why is this is only checking for e-?

Reply:
E_MINUS happens to be the slightly awkwardly named PDG code corresponding to the absolute value for electrons

Reply:
I see - thank you! How about photons?

Reply:
PFO convention is muon = track, electron = shower, so it's track/shower distinction rather than actual PID here.